### DISCUSSION TITLE
How to Find Best Resource for coding

### DISCUSSION THREAD

**Discussion by abhishek-nexgen-dev** [community/community#127628] [Programming Help]
### Body

I Expend my money to Purchase Some Cource But After Watching  That Cource I  realise This cource is not Pratical based so can you Suggest me how to find best Resource for Coding 

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by rashi24-p** [comment]
The best resources for coding depend on your skill level and area of interest. Like for learning any language you can prefer this: https://www.freecodecamp.org/ - and also read documentations .

**Comment by peterklein2024** [comment]
I had some serious issues with my site and my developer was unable to help so I approached Chatgpt and it was able to help with basic coding. My [podiatry](https://surgicalassociatespa.com/) treatment site was suffering loading issue.

**Reply by abhishek-nexgen-dev** [reply]
Thanka ❤️

**Comment by RafaelJohn9** [comment]
@Code-With-Abhishek-Kumar if you are new to programming best way to learn, is by first choosing a programming language, then  learning the basics, building basic projects and advancing to complex ones with the same programming language, I know this might be  a cliche statement but believe me the toil actually has to happen, you can use chatgpt only for almost impossible debugging, parts. There is no nice way to learn programming through courses that teach you everything, eventually you will forget, checkout this  ![repo](https://github.com/codecrafters-io/build-your-own-x)  for projects to create

**Comment by YuheshPandian** [comment]
## Resources(GitHub Repos) for Programmers

- [build-your-own-X](https://github.com/codecrafters-io/build-your-own-x)
- [ultimate-coding-resources](https://github.com/Dylan-Israel/ultimate-coding-resources)
- [the-book-of-secret-knowledge](https://github.com/trimstray/the-book-of-secret-knowledge)

**Reply by YuheshPandian** [reply]
From these repos you can navigate to the particular resource you need to complete your projects.

Thank you.

**Reply by YuheshPandian** [reply]
You can get additional help from LLMs like ChatGPT or Gemini.

**Reply by ebndev** [reply]
Hi @YuheshPandian, thanks for taking the time to answer!

@Code-With-Abhishek-Kumar when you're ready to move onto the GitHub part of your learning, please do explore and post any questions in Discussions! Best of luck on your coding journey 🌟
